content/applications/hr/referrals/alerts.rst

```diff
@@ -0,0 +1,56 @@
+======
+Alerts
+======
+
+In the *Referrals* application, it is possible to post a message, also referred to as an *alert*, at
+the top of the dashboard to share important information with users.
+
+Alerts appear on the main dashboard for the specified time configured on the individual alert.
+
+.. image:: alerts/alerts.png
+   :align: center
+   :alt: Two alert banners appear above the user's photo.
+
+Create an alert
+===============
+
+Only users with :guilabel:`Administrator` rights for the *Recruitment* application can create
+alerts. To add a new alert, navigate to the :menuselection:`Referrals app --> Configuration -->
+Alerts`.
+
+Click :guilabel:`Create` to open a blank alert form. Enter the following information on the form:
+
+- :guilabel:`Date From`: the date the alert starts. On this date, the alert is visible on the
+  dashboard.
+- :guilabel:`Date To`: the date the alert ends. After this date, the alert is hidden from view.
+- :guilabel:`Company`: the current company populates this field by default. To modify the company
+  the alert should be displayed for, select the company from the drop-down menu. If this field
+  remains blank, the alert is visible to everyone with access to the *Referrals* application. If a
+  company is specified, only users within that company (who also have access to the *Referrals*
+  application) will see the alert. This field only appears when in a multi-company database.
+- :guilabel:`Alert`: enter the text for the alert. This message appears inside the alert banner on
+  the main dashboard.
+- :guilabel:`On Click`: there are three options for the alert. Click the radio button next to the
+  desired selection. The options are:
+
+  - :guilabel:`Not Clickable`: the alert only displays text, there is no link to click.
+  - :guilabel:`Go to All Jobs`: the alert contains a link that, when clicked, navigates to the
+    website with all the currently posted job positions.
+  - :guilabel:`Specify URL`: the alert contains a link to a specific URL that, when clicked,
+    navigates to that URL. When selected, a :guilabel:`URL` field appears below the :guilabel:`On
+    Click` section. Enter the URL in the field.
+
+.. image:: alerts/alert-form.png
+   :align: center
+   :alt: An alert form completely filled in with all selections entered.
+
+Dismiss an alert
+================
+
+It is possible to dismiss an alert if a user does not wish to see a specific alert again.
+
+Click the :icon:`fa-times` icon on the far right side of the alert to remove the alert from the
```

Comment:
```suggestion
To dismiss an alert, click the :icon:`fa-times` :guilabel:`(x)` icon on the far-right side of the alert to remove the alert from the
```

Reply:
I agree with everything but the gulabel, since the times icon is an X- so it's just a bilder X next to a bold X, so I don't htink  it's needed.